Project2015To2017/Analysis/ExtensionMethods.cs

```diff
@@ -1,82 +0,0 @@
-using System;
```

Comment:
I had named all of these `ExtensionMethods` for consistency on the API. Probably not that big of a deal though since it’s all open source.

Reply:
Now there are:
* `P15To17.Analysis.AnalysisExtensions`
* `P15To17.Definition.ProjectExtensions`
* `P15To17.Extensions`

Seems pretty consistent and describing the purpose.

Reply:
Yeah, maybe went a bit overboard there. The initial comment here was more of a reason why I did it rather than it needs to be changed.

Reply:
Oh, I see.